Examples/Objective-C/ViewController.m

```diff
@@ -89,7 +89,7 @@ - (void)progressDidChange:(NSNotification *)notification {
     // If you are not using MapboxNavigationUI,
     // this would be a good time to update UI elements.
     // You can grab the current routeProgress like:
-    // let routeProgress = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationRouteProgressKey] as! RouteProgress
```

Comment:
it's not being removed, its being updated to the correct key since there is a slight name change in Constants.swift file